CMakeLists.txt

```diff
@@ -180,35 +180,15 @@ else()
 endif()
 
 if(NOT Protobuf_FOUND)
+  # prefer CONFIG mode over MODULE mode, there were a few CMake/protobuf fuckups
+  set(CMAKE_FIND_PACKAGE_PREFER_CONFIG ON)
```

Comment:
finally found a solution that should work: https://cmake.org/cmake/help/latest/variable/CMAKE_FIND_PACKAGE_PREFER_CONFIG.html. the intention was different for this, but it works for our case.